tools/automation/cli_linter/rules/linter_exclusions.yml

```diff
@@ -0,0 +1,42 @@
+---
+# this is an example linter exclusion file
+
+help-entry-name:
```

Comment:
Consider adding a comment of an example.
`help-entry-name` still doesn't tell me what to actually type.
Would it be what is in `helps['this text']` as `this text`?